app/views/notifications/_global_bar.html.erb

```diff
@@ -12,7 +12,7 @@
 -%>
 
 <% if show_global_bar %>
-  <% content_for :head do %>
+  <% content_for :body_end do %>
     <%= javascript_include_tag 'global-bar-init', integrity: false %>
```

Comment:
I think this was originally included right away to avoid any kind of flash of the banner before disappearing, e.g: if someone has manually hidden it or seen the banner 3 times already. Are we sure this change doesn't lead to that happening?

Reply:
As I'm reading the `global-bar-init.js` we show the banner when the script kicks in based on a set of conditions. Will double/triple check all scenarios to be sure.

Reply:
Tested before and after accepting cookies, showing it 3 times or manually hiding it. It's now [deployed in integration](https://www.integration.publishing.service.gov.uk). I don't see any change in behavior.

Reply:
It will become a component one day 😇 so we'll be more confident it works similarly with any other script.